### ISSUE TITLE
OSPF distribute-list

### ISSUE THREAD

**Issue by clxnetom** [bibigon812/bibigon812-quagga#30]
Didn't find any way to do distribute-lists in OSPF, maybe I'm just missing it or it is in the pipes but I managed to get it somewhat working (not sure about removing them): https://github.com/clxnetom/bibigon812-quagga/tree/ospf-distribute-list#ospf-distribute-lists

```
router ospf
distribute-list acl out kernel
```

**Comment by bibigon812** [comment]
```yaml
quagga::ospf::router:
  distribute-list:
    - <ACCESS_LIST_NAME> out <protocol>
```

**Comment by clxnetom** [comment]
```yaml
quagga::ospf::router:
  distribute-list:
    - acl out kernel
```
```json
{
    "project_page": "https://github.com/bibigon812/bibigon812-quagga",
    "source": "https://github.com/bibigon812/bibigon812-quagga.git",
    "summary": "Puppet module to manage Quagga",
    "version": "3.4.0"
}
```
```bash
Error: no parameter named 'distribute-list' at ... quagga/manifests/ospf.pp 17
```
```bash
Error: no parameter named 'distribute_list' at ... quagga/manifests/ospf.pp 17
```

**Comment by bibigon812** [comment]
This feature is not implemented yet. BTW, the last version is 3.5.0.

**Comment by clxnetom** [comment]
:+1: 

I tried an implementation using `quagga::router::ospf` but I couldn't figure out how to remove a distribute-list so I went down the other route (but still not sure if it will remove them). Looking forward to reading the new code once the feature is implemented.

**Comment by bibigon812** [comment]
@clxnetom, you may checkout PR#31

**Comment by bibigon812** [comment]
Released the 3.5.1
```
mod 'bibigon812-quagga', '3.5.1'
```
